Backpack-SwiftUI/Color/Classes/Generated/BPKStaticColors.swift

```diff
@@ -0,0 +1,240 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// Backpack Static Colors
+public extension BPKColor {
+    private init(red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat) {
+        self.init(value: UIColor(red: red, green: green, blue: blue, alpha: alpha))
+    }
+
+    /// The `backgroundAlternativeDarkColor` color from the Backpack palette - rgb(0, 0, 0)
+    static let backgroundAlternativeDarkColor = BPKColor(red:0.000, green:0.000, blue:0.000, alpha:1)
```

Comment:
one thing we did for compose was to exclude all of the dark/light colours as they should only really be used as dynamic colours - otherwise just use the sky blue or whatever directly